tests/core/test_models.py

```diff
@@ -138,5 +138,9 @@ def test_cache(self):
         ArticleRevision.objects.create(
             article=a, title="test", content="# header"
         )
-        self.assertEqual(a.get_cached_content(), """<h1 id="wiki-toc-header">header</h1>""")
-        self.assertEqual(a.get_cached_content(), """<h1 id="wiki-toc-header">header</h1>""")
+        expected_markdown = (
+            """<h1 id="wiki-toc-header">header"""
+            """<a class="article-edit" href="/1/_plugin/editsection/1-0-0/header/h/">[edit]</a></h1>"""
+        )
+        self.assertEqual(a.get_cached_content(), expected_markdown)
+        self.assertEqual(a.get_cached_content(), expected_markdown)
```

Comment:
This could change depending on the plugin set active. Could you instead test that the link is present in the `expected_html` (not markdown) ? You can use `self.assertIn`.

Reply:
markdown was a misnomer, copied from test_template_tags. Changed that here and
as well there.

assertIn() would not allow to check for a correct closing "\</h1>" tag. Because of
that I used now assertRegexpMatches() here and as well in test_template_tags.
That test relied as well on an active EditSection plugin.